File: web/js/Horario.js
 // funcion Horario
function Horario (id,fechSalida,horaSalida,fechLlegada,horaLlegada,precio){ 
  this.Horario(id,fechSalida,horaSalida,fechLlegada,horaLlegada,precio);
}

// prototype asociado a la funcion Persona
// solo metodos
Horario.prototype={ 
  Horario: function(id,fechSalida,horaSalida,fechLlegada,horaLlegada,precio){
	this.id=id;
	this.horaSalida=horaSalida;
    this.horaLlegada=horaLlegada;
	this.fechSalida=fechSalida;
	this.fechLlegada=fechLlegada;
	this.precio=precio;
  }
};

File: src/java/airline/model/Horario.java
package airline.model;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author John
 */
public class Horario implements Jsonable{
    String id;
    String fechSalida;
    String horaSalida;
    String fechLlegada;
    String horaLlegada;
    String precio;
    
    
    public Horario(String id, String fechSalida, String horaSalida, String fechLlegada, String horaLlegada, String precio){
        this.id=id;
        this.fechSalida = fechSalida;
        this.horaSalida = horaSalida;
        this.fechLlegada = fechLlegada;
        this.horaLlegada = horaLlegada;
        this.precio = precio;
        
    }
}
